apps/data/database/connect.sh

```diff
@@ -0,0 +1 @@
+PGPASSWORD=johuigytfrdcv psql -h localhost -p 8310 -d martletplace -U martletplace
```

Comment:
Typically database passwords are stored as environment variables in a .env file which aren't pushed to the repo